src/main/webapp/resources/js/pages/launch/LaunchForm.jsx

```diff
@@ -40,51 +39,26 @@ export function LaunchForm() {
       initialValues={state.initialValues}
     >
       <Row gutter={[16, 16]}>
-        <Col sm={24} md={19}>
+        <Col sm={24} md={18}>
           <Space direction="vertical" style={{ width: `100%` }}>
             <LaunchDetails />
+            <LaunchParameters form={form} />
             <SharePipelineResults />
             <ReferenceFiles />
-            <SavedParameters form={form} />
-            <ParameterWithOptions parameters={state.parameterWithOptions} />
             <LaunchFiles />
           </Space>
         </Col>
-        <Col sm={24} md={5}>
-          <Anchor>
```

Comment:
Just a question. Why did you get rid of the anchors? I liked how it looked and the functionality made sense

Reply:
As per our discussion on Friday, they were of now real benifit.